src/libs/ajax/workflows-app/Cbas.js

```diff
@@ -19,6 +19,11 @@ export const Cbas = (signal) => ({
       const res = await fetchFromProxy(cbasUrlRoot)('api/batch/v1/methods', _.mergeAll([authOpts(), jsonBody(payload), { signal, method: 'POST' }]));
       return res.json();
     },
+    delete: async (cbasUrlRoot, methodId) => {
+      const keyParams = qs.stringify({ method_id: methodId });
+      const res = await fetchFromProxy(cbasUrlRoot)(`api/batch/v1/methods?${keyParams}`, _.mergeAll([authOpts(), { signal, method: 'DELETE' }]));
```

Comment:
TOL (pedantic nit): Is `DELETE` the appropriate HTTP code? We're calling this delete to users but it's really a `PATCH` to set an `archive` field to true.

Reply:
good point. I like `PATCH` better, actually. then I can stop using the word "delete" in the API